### ISSUE TITLE
Deleting a single message (to trash) doesn't work

### ISSUE THREAD

**Issue by tenzap** [back2arie/Kalkun#217]
Deleting a single message (to trash) doesn't work (messag remains in the inbox)
Only deleting a conversation moves it to the trash

Same problem on inbox & outbot

**Comment by tenzap** [comment]
Could be related to https://github.com/bcit-ci/CodeIgniter/issues/5952

**Comment by john-arvid** [comment]
I tested this myself and it works. 
I tried to delete one message in a conversation that had two messages, the one I deleted went to the trash.
Anything special to reproduce?

**Comment by tenzap** [comment]
I'm using Postgresql, what about you ?
I will post the error that is in the logs

**Comment by tenzap** [comment]
```
ERROR - 2020-04-30 07:28:10 --> Severity: Warning --> pg_query(): Query failed: ERREUR:  la relation « sentitems&quot; as &quot;a&quot;, &quot;user_sentitems » n'existe pas
LINE 1: UPDATE &quot;sentitems&quot;&quot; as &quot;&quot;a&quot;&quot;, &quot;&quot;user_sentitems&quot; as &quot;b&quot; SET &quot;...
               ^ /var/www/kalkun/system/database/drivers/postgre/postgre_driver.php 242
ERROR - 2020-04-30 07:28:10 --> Query error: ERREUR:  la relation « sentitems" as "a", "user_sentitems » n'existe pas
LINE 1: UPDATE "sentitems"" as ""a"", ""user_sentitems" as "b" SET "...
               ^ - Invalid query: UPDATE "sentitems"" as ""a"", ""user_sentitems" as "b" SET "a"."id_folder" = '5', "b"."trash" = '1'
WHERE "a"."ID" = '373'
AND "a"."ID" =  "b"."id_sentitems"
```

Which says : relation/table « sentitems" as "a", "user_sentitems » doesn't exist

**Comment by john-arvid** [comment]
ok, I am using mysql.

**Comment by tenzap** [comment]
fixed with #242
